fractions/javaee/undertow/module.conf

```diff
@@ -3,3 +3,4 @@ org.wildfly.extension.undertow
 io.undertow.core
 io.undertow.servlet
 org.ow2.asm
+org.wildfly.swarm.security
```

Comment:
So, we have a trick, sorta, to avoid having to have mvn dependencies, but being able to have pseudo fraction optional dependencies.

if you inject

    Instance<Security> security

you can reference the superclass of the `SecurityFraction` and inject by it, without having to have enough Maven dependency to see `SecurityFraction`.  This works because the parent class of `SecurityFraction` is `Security` (I think) and exists in the config-api which is available across the board for all fractions.

So, while we actually will inject (if present) a `SecurityFraction`, you only see it by the one level up superclass of `Security` from config-api.